apps/fabric-website/src/components/App/AppState.tsx

```diff
@@ -88,6 +91,7 @@ export const AppState: IAppState = {
       title: 'Components',
       url: '#/components',
       className: 'componentsPage',
+      isTopNavHeader: true,
```

Comment:
Sorry to be pedantic, but I think something like `isTopNavItem` or `isTopNavLink` would be more clear. This makes it sound like the item itself is a top nav header 🙂

Reply:
You're right! I'll update.